features/webgl2-context-desynchronized.yml

```diff
@@ -0,0 +1,5 @@
+name: WebGL2 `desynchronized` parameter
```

Comment:
Has same support as WebGL1 `desynchronized` so we could merge these two. I feel like WebGL1/WebGL2 are very different and keeping them separate makes sense. But can be convinced otherwise.